adafruit_pybadger/pygamer.py

```diff
@@ -0,0 +1,123 @@
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""
+`adafruit_pybadger.pygamer`
+================================================================================
+
+Badge-focused CircuitPython helper library for PyGamer.
+
+
+* Author(s): Kattni Rembor
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+
+* `Adafruit PyGamer <https://www.adafruit.com/product/4277>`_
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+
+from collections import namedtuple
+import board
+import analogio
+import digitalio
+import audioio
+from gamepadshift import GamePadShift
+import adafruit_lis3dh
+from adafruit_pybadger.pybadger_base import PyBadgerBase
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_PyBadger.git"
+
+Buttons = namedtuple("Buttons", "b a start select")
+
+class PyGamer(PyBadgerBase):
+    """Class that represents a single PyGamer."""
+
+    _audio_out = audioio.AudioOut
+    _neopixel_count = 5
+
+    def __init__(self):
+        super().__init__()
+
+        i2c = board.I2C()
+
+        int1 = digitalio.DigitalInOut(board.ACCELEROMETER_INTERRUPT)
+        try:
+            self._accelerometer = adafruit_lis3dh.LIS3DH_I2C(i2c, address=0x19, int1=int1)
+        except ValueError:
+            self._accelerometer = adafruit_lis3dh.LIS3DH_I2C(i2c, int1=int1)
+
+        self._buttons = GamePadShift(digitalio.DigitalInOut(board.BUTTON_CLOCK),
+                                     digitalio.DigitalInOut(board.BUTTON_OUT),
+                                     digitalio.DigitalInOut(board.BUTTON_LATCH))
+
+        self._pygamer_joystick_x = analogio.AnalogIn(board.JOYSTICK_X)
+        self._pygamer_joystick_y = analogio.AnalogIn(board.JOYSTICK_Y)
+
+        self._light_sensor = analogio.AnalogIn(board.A7)
+
+    @property
+    def button(self):
+        """The buttons on the board.
+
+        Example use:
+
+        .. code-block:: python
+
+          from adafruit_pybadger import PyBadger
+
+          pybadger = PyBadger()
```

Comment:
I think some of the comments still have the old style example usage instead of the new way: 
`from adafruit_pybadger import pybadger`

pybadge and clue files have some as well this way.

Reply:
Nice catch

Reply:
Good catch! Thank you!